components/Chat/ChatMessages/ChatMessageMenu.vue

```diff
@@ -107,7 +107,7 @@
               class="mr-2"
               small
               @click="banUser"
-            >Ban</v-btn>
+            >Mute</v-btn>
```

Comment:
I still think 'master ignore list' is a better name. Might not be up to modern standards though

Reply:
This wasn't supposed to sneak into this PR, will revisit later on.